src/.editorconfig

```diff
@@ -295,6 +295,8 @@ dotnet_separate_import_directive_groups = false
 #### Naming styles ####
 
 dotnet_diagnostic.IDE1006.severity = error
+dotnet_diagnostic.CA1710.severity = error
```

Comment:
To be consistent with the current organization of the file, these should be in the `code quality analysis` section on line 21 that already has the rest of the configured CA rules.

Reply:
👍 done